pennylane_sf/simulator.py

```diff
@@ -198,6 +222,12 @@ def probability(self, wires=None):
             to the resulting probability. The dictionary should be sorted such that the
             state tuples are in lexicographical order.
         """
+        wires = wires or self.wires
```

Comment:
Just to make sure: by this point, it was already checked in the QNode that the labels make sense with regards to the labelling of the device, correct? :thinking:

Reply:
Yes, line 368 in base.py checks this. And it would be picked up by ``map_wires`` as well, although with a more specific error message.